backends-common/elasticsearch-v7/src/main/java/org/apache/james/backends/es/v7/NodeMappingFactory.java

```diff
@@ -54,15 +51,18 @@
     public static final String IGNORE_ABOVE = "ignore_above";
 
     public static ReactorElasticSearchClient applyMapping(ReactorElasticSearchClient client, IndexName indexName, XContentBuilder mappingsSources) throws IOException {
-        if (!mappingAlreadyExist(client, indexName)) {
+        createMapping(client, indexName, mappingsSources);
+
+        /*if (!mappingAlreadyExist(client, indexName)) {
             createMapping(client, indexName, mappingsSources);
-        }
+        }*/
         return client;
     }
 
+    //TODO: since ElasticSearch version 7 this check does not work anymore. Find another way to check whether mapping exists or not
     // ElasticSearch 6.3.2 does not support field master_timeout that is set up by 6.4.3 REST client when relying on getMapping
     @SuppressWarnings("deprecation")
-    public static boolean mappingAlreadyExist(ReactorElasticSearchClient client, IndexName indexName) throws IOException {
+    /*public static boolean mappingAlreadyExist(ReactorElasticSearchClient client, IndexName indexName) throws IOException {
         try {
             client.getLowLevelClient().performRequest(new Request("GET", "/" + indexName.getValue() + "/_mapping/"));
```

Comment:
https://www.elastic.co/guide/en/elasticsearch/reference/7.10/indices-get-mapping.html#get-mapping-api-query-params

maybe trying with putting `allow_no_indices` param to false?

On an other note as well, before the client for 6.3 didn't have a method to get the mapping of an indice, which is why we used the low level API. The 7.10 has a method it seems though: https://www.elastic.co/guide/en/elasticsearch/client/java-rest/7.10/java-rest-high-get-mappings.html

You might want to check it out too :)